src/manager/layout.js

```diff
@@ -193,7 +194,9 @@ export function openAddonBrowser() {
   layout.addonBrowserOpen = true;
 }
 export function hideAddonBrowser() {
+  console.log("hide addon browser");
```

Comment:
Do we want to remove console logs before prod?

Reply:
Good spot.